src/main/kotlin/no/nav/familie/baks/mottak/task/VurderLivshendelseService.kt

```diff
@@ -329,24 +283,36 @@ class VurderLivshendelseTask(
         }
     }
 
-    private fun søkEtterÅpenOppgavePåAktør(
-        aktørId: String,
-        type: VurderLivshendelseType,
-    ): Oppgave? {
-        val vurderLivshendelseOppgaver = oppgaveClient.finnOppgaverPåAktørId(aktørId, Oppgavetype.VurderLivshendelse)
-        secureLog.info("Fant følgende oppgaver: $vurderLivshendelseOppgaver")
-        return vurderLivshendelseOppgaver.firstOrNull {
-            it.beskrivelse?.startsWith(type.beskrivelse) == true && (
-                it.status != StatusEnum.FERDIGSTILT || it.status != StatusEnum.FEILREGISTRERT
-            )
+    private fun hentRestFagsak(
+        fagsakId: Long,
+        tema: Tema,
+    ): RestFagsak {
+        return when (tema) {
+            Tema.BAR -> baSakClient.hentRestFagsak(fagsakId)
+            Tema.KON -> ksSakClient.hentRestFagsak(fagsakId)
+            Tema.ENF, Tema.OPP -> throw RuntimeException("Tema $tema er ikke støttet")
+        }.also {
+            secureLog.info("Hentet rest fagsak: $it, tema: $tema")
         }
     }
 
+    private fun hentSisteBehandlingSomErIverksatt(restFagsak: RestFagsak): RestUtvidetBehandling? {
+        return restFagsak.behandlinger
+            .filter { it.steg == STEG_TYPE_BEHANDLING_AVSLUTTET }
```

Comment:
Ikke din feil, men her tror jeg vi har en feil. Burde være som i ba-sak. Burde filtrere bort henlagte som i ba-sak. Og Vedtatte er mer riktig enn iverksatte

private fun List<Behandling>.hentSisteSomErVedtatt() =
        filter { !it.erHenlagt() && it.status == BehandlingStatus.AVSLUTTET }
            .maxByOrNull { it.aktivertTidspunkt }

Reply:
Jeg tar det i en separat PR siden aktiverttidspunkt per i dag ikke kommer med restobjektet fra ba-sak